alerter.js

```diff
@@ -14,38 +14,42 @@
 
 // in case some invalid javascript was loaded before
 ;
-// alerter code, wrapped inside a self-executable anonymous funcation.
+// alerter code, wrapped inside a self-executable anonymous function.
 // undefined is passed to ensure it was not modified, as it's mutable on
 // some browsers
 (function (undefined) {
     "use strict";
 
     var defaults = {
-            // the height of the alert div
-            height: 50,
-            // the foreground and background colors for the alert
-            backgroundColor: 'A200FF',
-            foregroundColor: 'FFFFFF',
-            // font settings
-            fontFamily: 'Segoe UI',
-            fontSize: 13,
-            // probably the most interesting property you will change, the
-            // text of the div
+            id: undefined, // optional id if wanted
             text: 'Default Alert Text',
-            // default margin, padding and size
-            margin: 15,
-            padding: 5,
-            minWidth: 250,
-            // how long before hiding it, in seconds
-            duration: 2,
+            cssClassName: undefined,
+            styles: {
+                // the height of the alert div
+                height: '50px',
+                // the foreground and background colors for the alert
+                backgroundColor: '#A200FF',
+                color: '#FFFFFF',
+                // font settings
+                fontFamily: 'Segoe UI',
+                fontSize: '13px',
+                // default margin, padding and size
+                margin: '15px',
+                padding: '5px',
+                minWidth: '250px'
+            },
+            duration: 3,
+            // needed for stackable
             // these two options are for the fadeOut, and dictate how fast it is
             fadeStep: 5,
             fadeSpeed: 25,
             // show it bottom right or bottom left? 
-            orientation: 'right',
+            xOrientation: 'left', // left/right
```

Comment:
Let's default this to right

Reply:
Yes, "Mandatory", I don't know if it's the right term, but It's needed for the alerter to work, but as defaults values are defined, you don't need to define them in your config.
What word would you use ?

I'll make the update to the default config.